include/itkLinearAnisotropicDiffusionLBRImageFilter.hxx

```diff
@@ -325,7 +325,7 @@ LinearAnisotropicDiffusionLBRImageFilter< TImage, TScalar >
       !stencilIt.IsAtEnd();
       ++stencilIt, ++diagIt)
     {
-    for(auto i = 0; i < (int)StencilSize; ++i)
+    for(ImageDimensionType i = 0; i < (int)StencilSize; ++i)
```

Comment:
Why is StencilSize cast to `int`?

Reply:
I did not pay attention to the `StencilSize` variable and its casting. I cant have a look and possible remove it if unnecessary.

Reply:
They're declared as `unsigned`, and I'd assume they are casted to `int` so that the compiler does not complain. At first sight, it is weird, that it does not complain now since the image dimension is an `unsigned`. It did not complain in #29, but looks like I went too far in my changes. I'll amend my PR here to keep `i` an `unsigned` for the two cases reported. Thanks @dzenanz !

Reply:
Changed in 1eb6613. Thanks @dzenanz .